pydrive2/auth.py

```diff
@@ -308,18 +309,24 @@ def ServiceAuth(self):
         if set(self.SERVICE_CONFIGS_LIST) - set(self.client_config):
             self.LoadServiceConfigSettings()
         scopes = scopes_to_string(self.settings["oauth_scope"])
-        client_service_json = self.client_config.get("client_json_file_path")
-        creds_dict = self.client_config.get("client_creds_dict")
```

Comment:
Could introduce backwards compatibility for `client_creds_dict`, but I don't think it is worth it. It has been out for a day or so and was never documented.